src/studio/src/designer/backend/Services/Implementation/ConvertTextsService.cs

```diff
@@ -18,14 +18,16 @@ namespace Altinn.Studio.Designer.Services.Implementation
     public class ConvertTextsService : IConvertTextsService
     {
         private readonly IAltinnGitRepositoryFactory _altinnGitRepositoryFactory;
+        private readonly ITextsService _textsService;
 
         /// <summary>
         /// Constructor
         /// </summary>
         /// <param name="altinnGitRepositoryFactory">IAltinnGitRepository</param>
```

Comment:
Consider just removing the param tags, or add the missing one to get rid of the Sonar warning.